### PR TITLE
Rotational camera in simulation

### LINKED ISSUE #130: Create rotational stereo camera URDF
In order to make faster prototyping please create stereo camera using motor for rotation

### PR DESCRIPTION
**Pull request by MaxZhaloba**
Closes #130

### CODE REVIEW

**Comment by MaxZhaloba** [comment]
@AndriyPt, I've converted the joint for the head link to continuous type. It makes it possible to rotate the camera in Rviz. Please let me know if any other adjustments are necessary.

**Comment by AndriyPt** [comment]
@MaxZhaloba in real robot this link is fixed. Please connect "eyes" with extra bar and make it rotational on top of Г part of the robot.

File: leobot_description/robot/robot.urdf.xacro [hunk 1]
```diff
@@ -1,75 +1,73 @@
...
+  <joint name="neck_head_joint" type="fixed">
+    <origin rpy="${pi/2} ${pi/2} ${pi}" xyz="${-neck_length/2 + head_height/2} ${neck_height/2 + head_width/2} 0"/>
+    <parent link="neck_link"/>
+    <child link="head_link"/>
+  </joint>
 
-    <xacro:fish_camera prefix = "left" parent_link_name="head_link" parent_height="${head_height}" parent_width = "${head_width}" side="1"/>
-    <xacro:fish_camera prefix = "right" parent_link_name="head_link" parent_height="${head_height}"  parent_width = "${head_width}" side="-1"/>
+  <xacro:fish_camera prefix = "left" parent_link_name="head_link" parent_height="${head_height}" parent_width = "${head_width}" side="1"/>
+  <xacro:fish_camera prefix = "right" parent_link_name="head_link" parent_height="${head_height}"  parent_width = "${head_width}" side="-1"/>
```
Comment (ldemk) [inline leobot_description/robot/robot.urdf.xacro]: Consider renaming the camera. It is not a fish_eye camera any more.

**Comment by ldemk** [comment]
@MaxZhaloba Did You make changes to the gazebo file?

**Comment by MaxZhaloba** [comment]
I'm working on it right now. Implementing the changes requested by Andriy. He told me that it's ok to leave the fish eye cameras for now but attach them to rotating bar.

**Comment by MaxZhaloba** [comment]
I've fixed the indentation and deleted the xml declaration in all xacro files. No significant changes, only code formatting.

**Comment by MaxZhaloba** [comment]
@AndriyPt, Please review my updated implementation.

**Comment by AndriyPt** [comment]
@MaxZhaloba it is hard to see you changes. Please revert indentation fixes and post them as separate pull request. Please do one task in one pull request. It is harder to review code such pull requests.

**Comment by MaxZhaloba** [comment]
Ok, will do.

**Comment by MaxZhaloba** [comment]
@AndriyPt, done.

**Review by AndriyPt — APPROVED** [review]

**Pull request merged**